audius-cli

```diff
@@ -457,6 +456,11 @@ def launch(ctx, service, seed, chain, yes):
     launch_plugins(ctx)
 
     prune()
+    if service == "discovery-provider":
+        ctx.invoke(launch_chain)
+        ctx.invoke(launch_pedalboard)
+        if chain:
+            return
```

Comment:
moved below the prune as this was removing the containers, maybe notifications should be down here too?

Reply:
The prune should only clean up old things
https://github.com/AudiusProject/audius-docker-compose/pull/311/files#diff-574c8985009813d4c2084b3112e16001d296077f2cdb6b7e63e5fc0472d15249R189

Not sure about this change as it may leave remnant containers lying around. Is it possible the disk is super full b/c if you look at the prune command it will drop everything if we're at >95%